util/llpcEmuLib.h

```diff
@@ -42,6 +42,20 @@ class Function;
 class Module;
 } // llvm
 
+namespace std
+{
+    template<> struct hash<llvm::StringRef>
+    {
+        typedef llvm::StringRef argument_type;
+        typedef std::size_t result_type;
+        result_type operator()(argument_type const&
```

Comment:
Stay in one line. We allow 120 characters per line

->
result_type operator()(argument_type const& str) const noexcept